src/ol_infrastructure/applications/ocw_site/Pulumi.applications.ocw_site.Production.yaml

```diff
@@ -3,3 +3,7 @@ secretsprovider: awskms://alias/infrastructure-secrets-production  # pragma: all
 encryptedkey: AQICAHgQalNS7T35ZlcFdhF0QuKeiJAbXMUbm01pjGwHEsjRCgGPbSz7qllaLKSxEZcmDRxmAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMiW0X4xHd5cZ68j1uAgEQgDtqOisD1vkDetSKsy4Q4aHqZt4ZoWiScmYMkFGsRXTOCpzP2bU1U20TVIiN+hQrSWe60lqWycNfePaPtA==  # pragma: allowlist-secret
 config:
   aws:region: us-east-1
+  ocw_site:domains:
+  - draft.ocw.mit.edu
+  - live.ocw.mit.edu
```

Comment:
What would this be for? I thought you said we didn't need an origin server.

Reply:
No particular reason, just seemed logical to have a live domain to match with draft :shrug: Happy to remove it.

Reply:
I appreciate the consistency, so fine to keep it. 

But I think we should configure the ocw-studio app to point to "ocw.mit.edu" for linking to published versions. The current mismatch of "ocw-published" and "ocwnext" is confusing for users.

Reply:
:+1: